pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -797,12 +804,12 @@ func combineTaskRunAndTaskSpecAnnotations(pr *v1beta1.PipelineRun, pipelineTask
 	return annotations
 }
 
-func getPipelineRunTimeout(ctx context.Context, pr *v1beta1.PipelineRun) metav1.Duration {
+func getPipelineRunTimeout(ctx context.Context, pr *v1beta1.PipelineRun) time.Duration {
```

Comment:
We need one of these for TaskRun timeouts too, the same config is used for both (which seems sort of odd to me 🤔 ), so maybe we can just write one `GetTimeout(context.Context, *metav1.Duration) time.Duration` and share it in both places.

Reply:
I noticed there is a `GetTimeout()` above the lines in the taskrun reconciler, so I'm going so use that to handle defaulting, but it just uses the static default, not the configurable default.  This should probably be fixed as well?

Reply:
Alright fixed in a separate commit.